src/ui/src/components/workflows/WorkflowsGenerationLoader.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+	<div class="WorkflowsLifeLoading">
+		<div class="logoContainer">
+			<img src="../../assets/logo.svg" alt="Writer Framework logo" />
+			<WdsLoaderDots class="dot-loader" color="black" />
+			<div class="rotating-text">
+				<span
+					v-for="(text, index) in texts"
+					:key="index"
+					:class="{ active: currentTextIndex === index }"
+				>
+					{{ text }}
+				</span>
+			</div>
```

Comment:
It's better to use `<p>` for text. We have a reset style, so using `<p>` instead of `<div>` should not affect the style
```suggestion
			<p class="rotating-text">
				<span
					v-for="(text, index) in texts"
					:key="index"
					:class="{ active: currentTextIndex === index }"
				>
					{{ text }}
				</span>
			</p>
```